### PR TITLE
Fix test for RedBoxError react type violation

### PR DESCRIPTION
**Pull request by blackswanny**
test `fusion dev` top-level error failed due to react type violation

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,4 +3,4 @@ node_modules/
 .fusion/
 .nyc_output/
 yarn-error.log
-
+.idea/
```
Comment (KevinGrandon) [inline .gitignore]: Not sure if we want to include ide-specific files in our gitignore files - might be better to add this to your user directory? I believe you can have your own files listed in a global `~/.gitignore` file which would work for all projects.

File: build/server-error.js [hunk 2]
```diff
@@ -12,12 +12,21 @@ const React = require('react');
 const RedBox = require('redbox-react').RedBoxError;
 const ReactDOMServer = require('react-dom/server');
 
-function renderError(error /*: any */) {
+function renderError(error /*: any = {} */) {
```
Comment (KevinGrandon) [inline build/server-error.js]: What's the purpose for the `= {}` assignment here? If it's for a default assignment it seems like that should be outside of the comment?

File: .gitignore (hunk 1 above)
Comment (blackswanny) [reply .gitignore]: removed

File: build/server-error.js (hunk 2 above)
Comment (blackswanny) [reply build/server-error.js]: yes, default value.In this case the very last fallback for Object version of param will not faill if somebody tries to use undefined instead.
but outside of comment flow type check fails

**Pull request merged**